docs/release_notes.md

```diff
@@ -35,6 +36,15 @@
 - Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
+- Fixed a bug that occurred when data from the target mesh was missing. *AdonisFX-1183*
+- Fixed a bug that allowed to remove target geometries while painting Slide On Geometry constraints. *AdonisFX-1192*
+- Fixed a bug that allowed to open the AdonisFX Paint Tool for the final mesh of an AdnSkinMerge deformer. *AdonisFX-1198*
```

Comment:
Remove the one about #1192. If slide on geometry was not out yet, let's not say we messed up during development :)).

About #1198 I've added other comment. Let's generalize for all deformers.

Reply:
Right.

Reply:
Please remove this one (same reason)

Reply:
Removed.